rest/admin_api_test.go

```diff
@@ -1137,12 +1136,13 @@ func TestDBOnlineWithDelayAndImmediate(t *testing.T) {
 	json.Unmarshal(response.Body.Bytes(), &body)
 	assert.True(t, body["state"].(string) == "Online")
 
+	// Wait until after the 2 second delay, since the online request explicitly asked for a delay
```

Comment:
Should the above 500ms sleep and GET /db/ also be replaced w/ WaitForDBOnline?

Reply:
Yep, changing that